nixos/modules/services/hardware/nvidia-container-toolkit/default.nix

```diff
@@ -52,6 +52,15 @@
         '';
       };
 
+      device-name-strategy = lib.mkOption {
+        default = "index";
+        type = lib.types.enum [ "index" "uuid" "type-index" ];
+        description = ''
+          Specify the strategy for generating device names,
+          passed to nvidia-ctk cdi generate.
```

Comment:
Could you also add a sentence indicating that this will impact how you refer to the device with `nvidia.com/gpu=` in the container runtime?

Despite it might seem obvious, I think is worth stating it.

Reply:
good point, appended a sentence